net/openthread-br/patches/100-fix-gcc14-build.patch

```diff
@@ -0,0 +1,11 @@
+--- a/third_party/openthread/repo/third_party/mbedtls/repo/library/common.h
++++ b/third_party/openthread/repo/third_party/mbedtls/repo/library/common.h
```

Comment:
Would be good if this patch has header to be applied with `git am`. And also, I can not see that this patch is part of the openthread repository.

I see they are right now using https://github.com/openthread/openthread/blob/02390412cfa12c33df1493a887024940ff198f61/third_party/mbedtls/repo/library/common.h#L202 even for the master branch (https://github.com/openthread/ot-br-posix/commit/71b05fe78b9bcbda26b5eb456b350b91f33de633)

They should update their third-party mbedtls.

Reply:
This issue is from upstream mbedtls, it lasts even in recent release branch.

Reply:
I guess since upstream doesn't seem to care too much, and we already took the patch in OpenWrt base repo, we can just merge this?

Reply:
CI is passing.